### DISCUSSION TITLE
Android release dstribution

### DISCUSSION THREAD

**Discussion by GerbertBless** [slackhq/nebula#689] [Q&A] [answered]
Hey, 

I'm an android user, but I refuse to use google play. I can't find any apk in your releases. Is there a different distribution method you guys use where I can get the app from?

**Comment by tsugabloom** [comment]
You can use the Aurora Store as a Play Store client. It's available on F-Droid and doesn't need a Google account or any Google Play Services etc installed. I've had good success with it on both Micro-G Lineage and stricter operating systems like GrapheneOS.

Unfortunately the app hasn't been updated since June 8, 2021.

**Comment by johnmaguire** [comment, marked as answer]
Unfortunately we don't have plans to support other distribution platforms at this time. However, the source code is available at https://github.com/DefinedNet/mobile_nebula if you'd like to build the APK yourself.

**Comment by GerbertBless** [comment]
Thanks for the timely response and good options.

**Comment by noseshimself** [comment]
There is quite a number of tools for downloading APKs from the Google Play Store. You can use them to side-load the package later.

**Comment by noseshimself** [comment]
Just as an additional hint: It seems that Android applications that specify themselves as VPN (i.e. "thing that has full access to all network packets and may modify/reroute them") can't be easily side-loaded on ChromeOS anymore (they might stop working) so the play store might be the only future-proof distribution method.
